doc/api/net.md

```diff
@@ -874,7 +874,8 @@ Additional options:
   [`socket.setTimeout(timeout)`][] after the socket is created, but before
   it starts the connection.
 
-Here is an example of a client of the previously described echo server:
+Following is an example of a client of the echo server described
```

Comment:
Following -> Here (I'm not sure your text is grammatical, I think it would have to be "Following this is an ...", but just "Here" was fine.